garden/Manipulating_models.md

```diff
@@ -0,0 +1,156 @@
+# Manipulating Models
+
+This tutorial will walk you through using various plugins to assist model and scene manipulation in the Ignition GUI.
+
+The Transform Control plugin is a combination of keybindings and transform control options: selection, translation, rotation and snapping.
+This chart of [Ignition's keyboard shortcuts](hotkeys) may be helpful for this tutorial.
+
+The View Angle plugin allows you to quickly and easily adjust the direction from which your scene faces an entity in the simulation.
+
+The Component Inspector plugin displays a variety of object attributes.
+
+The Align Tool aligns entities along or about a specific entity's bounding box.
+
+## Prerequisites
+
+Start by getting Ignition up and running with a sample world:
+
+```bash
+ign sim shapes.sdf
+```
+
+The previous tutorial, [Understanding the GUI](GUI_tutorial), explains the basics of navigating the Ignition GUI.
```

Comment:
Should this link be `gui` like it is for Fortress? 

https://github.com/gazebosim/docs/blame/master/fortress/Manipulating_models.md#L22

```suggestion
The previous tutorial, [Understanding the GUI](gui), explains the basics of navigating the Ignition GUI.
```